pydis_site/apps/api/viewsets/bot/offensive_message.py

```diff
@@ -0,0 +1,66 @@
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework.filters import SearchFilter
+from rest_framework.mixins import (
+    CreateModelMixin,
+    DestroyModelMixin,
+    ListModelMixin,
+    UpdateModelMixin
+)
+from rest_framework.viewsets import GenericViewSet
+
+from pydis_site.apps.api.models.bot.offensive_message import OffensiveMessage
+from pydis_site.apps.api.serializers import OffensiveMessageSerializer
+
+
+class OffensiveMessageViewSet(
+    CreateModelMixin, ListModelMixin, DestroyModelMixin, UpdateModelMixin, GenericViewSet
+):
+    """
+    View providing CRUD access to reminders.
+
+    ## Routes
+    ### GET /bot/offensive-message
+    Returns all offensive messages in the database.
+
```

Comment:
Does this route take any parameters or are we always returning all the entries in the database?